src/Behat/Behat/Definition/Context/Attribute/DefinitionAttributeReader.php

```diff
@@ -26,12 +24,12 @@
 final class DefinitionAttributeReader implements AttributeReader
 {
     /**
-     * @var string[]
+     * @var array<class-string<Attribute\Definition>, class-string<Call\RuntimeDefinition>>
      */
-    private static $classes = array(
-        Given::class => 'Behat\Behat\Definition\Call\Given',
-        When::class  => 'Behat\Behat\Definition\Call\When',
-        Then::class  => 'Behat\Behat\Definition\Call\Then',
+    private static $attributeToCallMap = array(
```

Comment:
I renamed the property in the hope that this is a bit clearer than "classes". However, for me the type hint and (aliased) class references is readable enough, so if I can rollback the rename if desired.